vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java

```diff
@@ -73,8 +72,8 @@
             255, 255, 0.0);
 
     protected final static String DEFAULT_FONT_FAMILIES = "var(--lumo-font-family)";
-    protected static final Color LEGEND_TEXT_COLOR = new SolidColor(128, 128,
-            128);
+    protected static final Color LEGEND_TEXT_COLOR = new SolidColor(255, 255,
```

Comment:
Is the legend text on a dark background? This color is now a semi-transparent white.

Reply:
I believe it is on a full black background, the one in line 68:
```java
protected final static SolidColor BGCOLOR = new SolidColor(255, 255, 255);
```

Reply:
255,255,255 is white 😄

Reply:
Then maybe we should keep the text light gray or black.